pkg/flowaggregator/flowaggregator.go

```diff
@@ -613,8 +614,11 @@ func (fa *flowAggregator) flowExportLoopProxy(stopCh <-chan struct{}) {
 
 	updateCh := fa.updateCh
 	for {
+		fa.metricsMutex.Lock()
```

Comment:
It seems the loop will always be holding the lock and waiting for events: acquire lock -> wait for event -> handle event -> release lock -> acquire lock -> ...

Should we use `atomic.Int64` for the counters?